### ISSUE TITLE
Classification counts don't match up

### ISSUE THREAD

**Issue by wgranger** [zooniverse/galaxy-zoo-touch-table#113]
It's possible for a table to show 9/10 classifications on a classification summary, but only one answer recorded on the bar charts (eg. one smooth, zero features or not a galaxy). This is likely an issue with counts in the local db vs. those coming from Caesar, etc. It will take a bit of investigation to determine what exactly is going on.

**Comment by wgranger** [comment]
Note, this was partially an error when updating the database with new subjects. In doing so, I cleared the smooth, features, and star (all answers) counts from the preexisting subjects. After that, those subjects contain a valid `classifications_count` from the data export, but no counts for each answer.

Answers should've been filled in from Caesar, but the Caesar production service may currently be down, leading the app to fall back on the incomplete database. 

In a nutshell, this isn't a huge problem data-wise. It's mainly a problem with the user seeing an inconsistent UI.

**Next steps:**
- Determine why/if the Caesar graphQL endpoint is down
- In the future aim to update the DB without clearing counts and update the docs on how to do this

**Comment by camallen** [comment]
Caesar graphql endpoints can be tested via instrospection cmd to determine what is available, e.g. 
```
curl -v -d '{"query": "{__schema {queryType {name fields {name}}mutationType {name fields {name}}}}"}' -H "Content-Type: application/json" -X POST https://caesar.zooniverse.org/graphql
# Switch the end points to test staging vs production 
```
You should see a result like
``` 
{
    "data": {
        "__schema": {
            "queryType": {
                "name": "QueryRoot",
                "fields": [
                    {
                        "name": "me"
                    },
                    {
                        "name": "workflow"
                    }
                ]
            },
            "mutationType": {
                "name": "MutationRoot",
                "fields": [
                    {
                        "name": "createDataRequest"
                    },
                    {
                        "name": "extractSubject"
                    },
                    {
                        "name": "upsertExtract"
                    },
                    {
                        "name": "upsertReduction"
                    }
                ]
            }
        }
    }
}
```
I'm happy to help debug any graphql issues

**Comment by wgranger** [comment]
@camallen I can replicate the above and get the expected response, but I'd ideally like to test a query similar to what the touch table app is sending. I'm not sure if that's done via introspection or can be done similarly through curl. Postman could be an option as well?

**Comment by camallen** [comment]
Gotcha - you can use curl / postman to recreate the query your app will send. 

If the introspection query is working then you know the Graphql system on the application is working at a basic level. Can you post some code links to the graphql query your app will send to caesar?

**Comment by wgranger** [comment]
Yep. Here's a link to the query in code. https://github.com/zooniverse/galaxy-zoo-touch-table/blob/master/GalaxyZooTouchTable/GalaxyZooTouchTable/Services/GraphQLService.cs#L20

Plugging in the workflow id `11308` and subject id `30091427` should provide results via production.

**Comment by wgranger** [comment]
@camallen Whoops. I think this turned out to be an issue with public reductions turned off. I turned it back on and am not getting a valid response.

Which means the touch table was running well without being connected to Caesar since the local database was doing all the heavy lifting. The issue was only noticed now that new subjects were added to the local db.
